example/src/screens/GenericClientScreen.tsx

```diff
@@ -142,15 +145,46 @@ export default function GenericClientScreen({
 }: GenericClientScreenProps) {
     const { client } = route.params;
 
-    const [method, setMethod] = useState("GET");
+    const [method, setMethod] = useState(METHOD.GET);
     const [url, setUrl] = useState("http://google.com"); //'https://jsonplaceholder.typicode.com/todos/1');
+    const [timeoutInterval, setTimeoutInterval] = useState(30);
     const [body, setBody] = useState("");
     const [requestHeaders, setRequestHeaders] = useState([
         { key: "", value: "" },
     ]);
     const [response, setResponse] = useState<ClientResponse>();
+    const [retryPolicyConfiguration, setRetryPolicyConfiguration] = useState<
+        RetryPolicyConfiguration
+    >({
+        type: undefined,
+        retryLimit: 2,
+        exponentialBackoffBase: 2,
+        exponentialBackoffScale: 0.5,
+    });
+
     const scrollView = useRef<ScrollView>(null);
```

Comment:
Just based on the last comment, it might be worth being explicit here calling it a `scrollViewRef`.